include/nil/blueprint/parser.hpp

```diff
@@ -1453,7 +1461,10 @@ namespace nil {
                             ASSERT(call_stack.size() == 0);
                             finished = true;
 
-                            if(PrintCircuitOutput) {
+                            if(print_output_format != no_print) {
+                                if(print_output_format == hex) {
+                                    std::cout << std::hex;
```

Comment:
I would suggest moving this to the logger. This will be easier to support latter